src/main/java/org/scoverage/plugin/SCoverageCheckMojo.java

```diff
@@ -201,9 +252,65 @@ else if ( coverage.statementCoveragePercent() < minimumCoverage )
 
     // Private utility methods
 
-    private boolean is100( Double d )
+    private static boolean is100( Double d )
     {
         return Math.abs( 100 - d ) <= 0.00001d;
     }
 
-}
\ No newline at end of file
+    private static boolean checkCoverage( Log logger, String metric, CoverageMetrics metrics,
+                                          double minimumStmt, double minimimBranch, boolean logSuccessInfo )
+    {
+        boolean stmt = checkCoverage( logger, "Statement:" + metric,
+                                      minimumStmt, metrics.statementCoveragePercent(), logSuccessInfo );
+        boolean branch = checkCoverage( logger, "Branch:" + metric,
+                                        minimimBranch, metrics.branchCoveragePercent(), logSuccessInfo );
+        return stmt && branch;
+    }
+
+    private static boolean checkCoverage( Log logger, String metric,
+                                          double minimum, double actual, boolean logSuccessInfo )
+    {
+        if ( minimum <= 0 )
+        {
+            return true;
+        }
+
+        if ( is100( minimum ) && is100( actual ) )
+        {
+            logSuccess( logger, String.format( "Coverage is 100%: %s!", metric ), logSuccessInfo );
```

Comment:
```suggestion
            logSuccess( logger, String.format( "Coverage is 100%%: %s!", metric ), logSuccessInfo );
```
otherwise it fails with `java.util.UnknownFormatConversionException: Conversion = ':'`

would be nice to add module3 to your awesome tests checking this branch )

Reply:
will do